internal/sync/sync.go

```diff
@@ -76,23 +82,18 @@ func (c *Command) Main(ctx context.Context, pwd string, args []string) error {
 		return err
 	}
 
-	url := os.Getenv(DepbotServerAddr)
-	if url == "" {
-		url = "http://app.depbot.com/api/sync"
-	}
-
 	if c.client == nil {
 		c.client = new(http.Client)
 	}
 
-	req, err := http.NewRequest(http.MethodPost, url, bytes.NewBuffer(jm))
+	req, err := http.NewRequest(http.MethodPost, serverURL(), bytes.NewBuffer(jm))
 	if err != nil {
 		return fmt.Errorf("error creating new request %w", err)
 	}
 
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", apiKey))
-	req.Header.Set("X-Revision-Hash", hash)
```

Comment:
@bcuadrad1 
Could we make this change in the FindLatestHash function?

Reply:
Yes. ill do it